c/tests/test_tables.c

```diff
@@ -5273,8 +5271,8 @@ test_dump_fail_no_file(void)
     errno = 0;
 
     /* Trying to dump without first sorting fails */
```

Comment:
We should be able to dump and load the unsorted tables without problems. Can just change the test to asserting ret is 0.

Reply:
I think this is taken care of.